### PR TITLE
Ap ruby tut fix

### PR DESCRIPTION
**Pull request by diabolo**
I'd review this one commit at a time rather than looking at overall diff. Each commit is trying to fix a specific small issue with the tutorial

### CODE REVIEW

File: content/10-minute-tutorial.md [hunk 1]
```diff
@@ -626,9 +629,10 @@ When("I ask whether is's Friday yet") do
   @actual_answer = is_it_friday?(@today)
 end
 
-Then("I should be told {string}") do |expected_answer|
+Then("I should be told {string}§:w") do |expected_answer|
```
Comment (olleolleolle) [inline content/10-minute-tutorial.md]: `§:w` is that slang for "I am leaving the vi editor"?

**Review by olleolleolle — COMMENTED** [review]
Thanks for maintaining the Ruby side of the tutorial! 

These are all good changes, but I had one question about a tiny thing, which looked like a typo.

**Comment by mlvandijk** [comment]
I approve the 2nd & 3rd commits, but have no opinion on the Ruby changes. :)
If what looks like a typo is fixed, I'd be happy to merge!

File: content/10-minute-tutorial.md (hunk 1 above)
Comment (olleolleolle) [reply content/10-minute-tutorial.md]: @diabolo Would you be OK with me editing this, and then merging?

Comment (aslakhellesoy) [reply content/10-minute-tutorial.md]: Just go ahead @olleolleolle

**Comment by mlvandijk** [comment]
Thanks @olleolleolle

**Pull request merged**